### ISSUE TITLE
Update never completes

### ISSUE THREAD

**Issue by perry-mitchell** [buttercup/buttercup-desktop#732]
Updates on mac from 1.13 -> 1.14 are hanging and never completing:

![image](https://user-images.githubusercontent.com/3869469/50712862-baa66680-107b-11e9-9a23-8eddb32aea35.png)

**Comment by perry-mitchell** [comment]
The log gives the reason:

```
[2019-01-04 23:21:10:0692] [info] Checking for update
[2019-01-04 23:21:12:0168] [info] Found version 1.14.0 (url: Buttercup-1.14.0-mac.zip, Buttercup-1.14.0.dmg)
[2019-01-04 23:21:13:0876] [info] Downloading update from Buttercup-1.14.0-mac.zip, Buttercup-1.14.0.dmg
[2019-01-04 23:21:13:0950] [info] / requested
[2019-01-04 23:21:13:0959] [info] /app.zip requested
[2019-01-04 23:21:14:0381] [info] download-progress listener count: 1
[2019-01-04 23:21:14:0381] [info] contentLength: 57686193
[2019-01-04 23:21:36:0122] [warn] Error: Code signature at URL file:///Users/perry/Library/Caches/pw.buttercup.desktop.ShipIt/update.dM1MrBV/Buttercup.app/ did not pass validation: code failed to satisfy specified code requirement(s)

[2019-01-04 23:21:36:0122] [error] Error: Error: Code signature at URL file:///Users/perry/Library/Caches/pw.buttercup.desktop.ShipIt/update.dM1MrBV/Buttercup.app/ did not pass validation: code failed to satisfy specified code requirement(s)
[2019-01-04 23:21:39:0993] [info] Proxy server for native Squirrel.Mac is closed (was started to download https://github.com/buttercup/buttercup-desktop/releases/download/v1.14.0/Buttercup-1.14.0-mac.zip)
```

**Comment by perry-mitchell** [comment]
Seems to be exactly this: https://github.com/Squirrel/Squirrel.Mac/issues/160

Now that we're publishing from my machine, the certificate is different and that's why updates are failing. Trying to use the old cert now.

**Comment by perry-mitchell** [comment]
Ok, v1.14 has been updated and it seems to work fine now. Seems that the update process (Squirrel) is actually quite secure (we can't even force a change in certificate 😅). We've used the original cert for now but will migrate again later when we figure out how to bridge it.

**Comment by julianpoemp** [comment]
I still have this problem when upgrading from version 1.14.0 to 1.15.0 on MacOS 10.14.3. Perhaps it's solved with 1.15.0?

**Comment by perry-mitchell** [comment]
@julianpoemp That should be the case yes..
